explorer/src/lib/dusk/components/card/Card.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+  import Icon from "../icon/Icon.svelte";
+  import Switch from "../switch/Switch.svelte";
+  import { makeClassName } from "$lib/dusk/string";
+
+  import "./styles.css";
+
+  /** @type {string} */
+  export let tag = "div";
+
+  /** @type {string | Undefined} */
+  export let className = undefined;
+
+  /** @type {string | Undefined} */
+  export let iconPath = undefined;
+
+  /** @type {string} */
+  export let heading;
+
+  export let hasToggle = false;
+  export let isToggled = false;
+
+  export let onSurface = false;
```

Comment:
I find this naming confusing as it resembles an event handler.
Being a boolean I would go for `isOnSurface` (same applies to the Switch).

Having said that, not necessary to do it now as it requires some thought, I see these things more as variants: `isOnSurface` is very specific and handles only one case.
We should open an issue to setup the variants we need.

Reply:
I think all issues regarding components that are migrated should be addressed in the `duskit` repo and not here. If there is something that could be improved regarding structure or naming conventions an issue should be opened.

Reply:
> Interesting you bring this up now, as this was approved some time ago in it's current form.

Well, I'm noticing only now so I brought up the issue.

Reply:
> I think all issues regarding components that are migrated should be addressed in the `duskit` repo and not here. If there is something that could be improved regarding structure or naming conventions an issue should be opened.

Fine by me.